clipboard/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@capacitor/clipboard",
+  "version": "0.0.1",
+  "description": "The Clipboard API enables copy and pasting to/from the clipboard.",
+  "main": "dist/plugin.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "scripts": {
+    "lint": "npm run prettier -- --check && npm run swiftlint -- lint",
+    "fmt": "npm run prettier -- --write && npm run swiftlint -- autocorrect",
```

```suggestion
    "fmt": "npm run prettier -- --write && npm run swiftlint -- autocorrect --format",
```
missing --format param